app/settings3.html

```diff
@@ -36,10 +36,12 @@ <h2 data-i18next="settings3.miscellaneous"></h2>
       <h2 data-i18next="settings3.language"></h2>
       <div class="">
         <select name="language" id="language"> <!--Supplement an id here instead of using 'name'-->
-        <option value="en">English</option>
-        <option value="sk" selected>Slovenčina</option>
-        <option value="fr">Français</option>
-      </select>
+          <!--Keep languages in the alphabet order-->
+          <option value="cs">Čeština</option>
+          <option value="en">English</option>
+          <option value="fr">Français</option>
+          <option value="sk" selected>Slovenčina</option>
```

Comment:
Could you please remove `selected` while you're at it? I guess that's something left from testing...

Reply:
Done.